### DISCUSSION TITLE
Where can I find the log info?

### DISCUSSION THREAD

**Discussion by iloug** [tauri-apps/tauri#5160] [General]
Hi, I have develop and publish a tauriapp to my client, On almost all windows 7,Windows 10,and 11, works perfect.

but on a Windows 10 Home edition computer, after double click the app, the mouse pointer just show busy one or two times and stop, I check the process manager, I can see my app and webview runtime are running,  but the main window not show.

I have uninstall the webview2 runtime and reinstall, also make a 32-bit version for it, but can not work.

I want know, if the tauri app can not run normal, where can I find error logs to check?

Thanks a lot.

**Comment by FabianLars** [comment]
It's windows so there probably are no logs (from the system and the webview). You can check Tauri's/Rust's panics by removing the first 4 lines of the main.rs (something with `windows_subsystem`) and running the app from within a Terminal. I doubt that there will anything useful tho since your app keeps running according to the task manager.
